### DISCUSSION TITLE
How to compile to .pyc

### DISCUSSION THREAD

**Discussion by RawEnchilada** [erg-lang/erg#196] [Q&A] [answered]
I've got a simple .er file that I'm trying to compile to .pyc.
The erg argument "--dump-as-pyc" seemingly does nothing, and documentation on how to use the available tools is pretty scarce.

**Comment by mtshiba** [comment, marked as answer]
There was a bug in the compiler and command line option parser that prevented the option from working properly.
This has been fixed in the latest version (v0.5.8).

**Reply by mtshiba** [reply]
And see `erg --help` or `erg --mode --help` for erg command options.

Tools such as package managers and formatters have not yet been implemented.
